main.js

```diff
@@ -115,37 +117,87 @@ function currencyFormat(price) {
   return '$' + priceStr
 }
 
-function createCard(item) {
-  return createElement('div', { class: 'card', style: 'width: 18rem;' }, [
+function renderCard(item) {
+  return createElement('div', { class: 'card', style: 'width: 18rem;', 'data-item-id': item.itemId }, [
     createElement('img', { class: 'card-img-top', src: item.imageUrl, alt: 'Card image cap' }, []),
     createElement('div', { class: 'card-body' }, [
       createElement('h5', { class: 'card-title' }, [item.brand]),
-      createElement('p', { class: 'card-text' }, [item.name]),
+      createElement('p', { class: 'card-text item-name' }, [item.name]),
       createElement('p', { class: 'card-text' }, [currencyFormat(item.price)])
     ])
   ])
 }
 
-function createCatalog(catalog) {
+function renderCatalog(catalog) {
+  var $container = createElement('div', { class: 'container' }, [])
+  var $rows = {}
+  var rowIndex = 0
+  $rows[rowIndex] = createElement('div', { class: 'row' }, [])
+
+  catalog.items.forEach((item, index) => {
```

Comment:
You shouldn't need to create multiple rows when using Bootstrap grids. If you choose your column classes carefully, the columns will wrap automatically.
https://getbootstrap.com/docs/4.0/layout/grid/#responsive-classes